ecosystem/controllers/runner.py

```diff
@@ -135,3 +135,38 @@ def workload(self) -> Tuple[str, List[CommandExecutionSummary]]:
             self.logger.warning("There in no terra version file...")
 
         return terra_version, [tox_tests_res]
+
+
+class PythonStyleRunner(Runner):
+    """Runners for styling Python repositories."""
+
+    def __init__(self,
+                 repo: Union[str, Repository],
+                 working_directory: Optional[str] = None,
+                 ecosystem_deps: Optional[List[str]] = None,
+                 repo_config: Optional[RepositoryConfiguration] = None):
+        super().__init__(repo=repo,
+                         working_directory=working_directory)
+        self.ecosystem_deps = ecosystem_deps or ["qiskit"]
+        self.repo_config = repo_config
+
+    def workload(self) -> List[CommandExecutionSummary]:
+        """Runs styles checks for python repository.
+
+        Steps:
+        - optional: check for .pylintrc file
+        - run lint
+        - form report
+
+        Returns: execution summary of steps
+        """
+        # check for existing .pylintrc file
```

Comment:
We need to use repo config in here like we are doing in tox checks to get configuration and render file. 
It's not ideal implementation right now, but we will refactor it later.

Reply:
Ok I see what you mean, I remember I got the doubt when coding it